packages/react-app/src/CreateInk.js

```diff
@@ -320,370 +322,417 @@ useEffect(() => {
-    if(canvasDisabled){
-      console.log("Canvas disabled")
-    } else {
-      saveDrawing(drawingCanvas.current, false)
+  let top, bottom, canvas, shortcutsPopover
+  if (props.mode === "edit") {
+
+    top = (
+        <div style={{ margin: "0 auto", marginBottom: 16}}>
+
+
+
+          <Form
+              layout={'inline'}
+              name="createInk"
+              //initialValues={{ limit: 0 }}
+              onFinish={createInk}
+              onFinishFailed={onFinishFailed}
+              labelAlign = {'middle'}
+              style={{justifyContent: 'center'}}
+          >
+
+            <Form.Item >
+
+            </Form.Item>
+
+            <Form.Item
+                name="title"
+                rules={[{ required: true, message: 'What is this work of art called?' }]}
+            >
+              <Input placeholder={"name"} style={{fontSize:16}}/>
+            </Form.Item>
+
+            <Form.Item
+                name="limit"
+                rules={[{ required: true, message: 'How many inks can be minted?' }]}
+            >
+              <InputNumber placeholder={"limit"}
+                           style={{fontSize:16}}
+                           min={0}
+                           precision={0}
+              />
+            </Form.Item>
+
+            <Form.Item >
+              <Button loading={sending} type="primary" htmlType="submit">
+                Ink!
+              </Button>
+            </Form.Item>
+          </Form>
+
+          <div style={{marginTop: 16}}>
+            <Tooltip title="save to local storage">
+              <Button
+                  disabled={canvasDisabled||drawingCanvas.current&&!drawingCanvas.current.lines.length}
+                  onClick={() => {
+                    if (canvasDisabled || drawingCanvas.current&&!drawingCanvas.current.lines) return;
+                    saveDrawing(drawingCanvas.current, true)
+                    console.log('drawing canvas current', drawingCanvas.current)
+                  }}><SaveOutlined /> {`${!drawingSaved?'SAVE *':'SAVED'}`}</Button>
+            </Tooltip>
+            <Button
+                disabled={canvasDisabled||drawingCanvas.current&&!drawingCanvas.current.lines.length}
+                onClick={() => {
+                  if (canvasDisabled || drawingCanvas.current&&!drawingCanvas.current.lines) return;
+                  undo()
+                }}><UndoOutlined /> UNDO</Button>
+            <Popconfirm
+                title="Are you sure?"
+                onConfirm={() => {
+                  if (canvasDisabled || drawingCanvas.current&&!drawingCanvas.current.lines) return;
+                  drawingCanvas.current.clear()
+                  //setLoadedLines()
+                  props.setDrawing()
+                }}
+                okText="Yes"
+                cancelText="No"
+            >
+              <Button
+              ><ClearOutlined /> CLEAR</Button>
+            </Popconfirm>
+            <Button
```

Comment:
We may want to move these to be next to the "Drafts" buttons (see https://github.com/BuidlGuidl/nifty-ink/pull/35)
I think probably doesn't make sense mixed in with all the drawing tools?